core/src/main/scala/org/apache/spark/util/SizeEstimator.scala

```diff
@@ -49,6 +49,11 @@ private[spark] object SizeEstimator extends Logging {
   private val FLOAT_SIZE   = 4
   private val DOUBLE_SIZE  = 8
 
+  // Fields can be primitive types, sizes are: 1, 2, 4, 8. Or fields can be pointers. The size of 
+  // a point is 4 or 8 depends on the JVM (32-bit or 64-bit) and UseCompressedOops flag.
```

Comment:
Nits:

size of point -> size of a pointer
depends on the JVM -> depending on the JVM